torch/_inductor/graph.py

```diff
@@ -1244,24 +1244,29 @@ def debug(msg):
                 strides = n.meta["val"].stride()
                 dense = torch._prims_common.is_non_overlapping_and_dense(n.meta["val"])
                 unbacked_symbols_in_strides = len(free_unbacked_symbols(strides)) > 0
+
                 # requiring a stride order for a non-dense output wouldn't
                 # recreate the same strides, and would fail with view, defer for now.
-                if not unbacked_symbols_in_strides and dense and len(strides):
-                    stride_order = ir.get_stride_order(strides)
-                    if (
-                        len(result.get_size()) == 4
-                        and n in self.nodes_prefer_channels_last
-                        and n.name not in self.user_visible_outputs
-                        and not is_input_for_as_strided
-                    ):
-                        stride_order = ir.NHWC_STRIDE_ORDER
-
+                if not unbacked_symbols_in_strides and len(strides):
+                    stride_order = []
+                    if dense:
+                        stride_order = ir.get_stride_order(strides)
+                        if (
+                            len(result.get_size()) == 4
+                            and n in self.nodes_prefer_channels_last
+                            and n.name not in self.user_visible_outputs
+                            and not is_input_for_as_strided
+                        ):
+                            stride_order = ir.NHWC_STRIDE_ORDER
                     allow_padding = (
                         n.name not in self.user_visible_outputs
                         and not is_input_for_as_strided
                     )
                     result = ir.ExternKernel.require_stride_order(
-                        result, stride_order, allow_padding=allow_padding
+                        result,
+                        stride_order if stride_order else None,
+                        allow_padding=allow_padding,
+                        actual_strides=strides,
```

Comment:
Need to figure out what should be the correct strides. Maybe only enable it for some specific operations.

Reply:
For this PR, it only focus on output node. So, we are safe.